src/display/text_layer.js

```diff
@@ -607,6 +607,9 @@ var renderTextLayer = (function renderTextLayerClosure() {
       }
       this._textDivProperties.set(textDiv, textDivProperties);
       this._container.appendChild(textDiv);
+      if (window.chrome) {
```

Comment:
Generally speaking, we don't want these sort of browser-specific compatibility hacks in the main code-base for reasons of e.g. readability/maintainability/performance.